src/button/interfaces.ts

```diff
@@ -124,11 +91,6 @@ export interface ButtonProps extends BaseComponentProps {
    */
   variant?: ButtonProps.Variant;
```

Comment:
Nit: this could be in the Button interfaces instead of here as ToggleButton overrides it with different values

Reply:
updated